test/studies/lcals/LCALSMain.skipif

```diff
@@ -0,0 +1,3 @@
+# This test takes too long if --fast is not used so only run it for
```

Comment:
Ah ok. I think it would be nice to do some correctness testing on these even if we have to throw --no-checks or something to start. That certainly doesn't have to be done prior to committing though